package.json

```diff
@@ -43,7 +43,6 @@
   "dependencies": {
     "fancy-log": "^1.3.3",
     "plugin-error": "^1.0.1",
-    "postcss": "^7.0.14",
     "postcss-load-config": "^2.0.0",
```

Comment:
Hm. Seems like we will need to update `postcss-load-config` or it will not be able to load PostCSS 8 plugins.

Reply:
Do you mean updating its version number in the `package.json`, or actually opening a PR against it?

Reply:
Open PR there

Reply:
https://github.com/michael-ciniawsky/postcss-load-config/pull/200